### ISSUE TITLE
Memory leak when using GIMP in multiple windows mode

### ISSUE THREAD

**Issue by robin92** [linuxmint/Cinnamon#4212]
When running GIMP in multiple windows (not single-window mode, must be turn on in multi-window mode) cinnamon leaks memory. Leak is caused by moving your mouse (painting) and a few lines leak ~8 MB!

I'm running latest Arch Linux with the following cinnamon packages:
```
cinnamon 2.4.8-1
cinnamon-control-center 2.4.2-3
cinnamon-desktop 2.4.2-1
cinnamon-menus 2.4.1-1
cinnamon-screensaver 2.4.2-1
cinnamon-session 2.4.3-2
cinnamon-settings-daemon 2.4.3-1
cinnamon-translations 2.4.4-1
```

This is a high severity bug mainly due to insane amounts of memory leaking in time.

**Comment by JosephMcc** [comment]
On Mint 17.1 Cinnamon 2.6. Opened gimp in multi-window mode, created a new file. Grabbed a paintbrush and started painting. Cinnamons memory usage holds steady. Certainly don't see the issue you're having.

**Comment by robin92** [comment]
Reproduced on all my computers running mentioned above software. I will compile Cinnamon 2.6 and give it a try.

**Comment by robin92** [comment]
Seems fixed in Cinnamon 2.6
